dms/models/dms_file.py

```diff
@@ -34,24 +34,21 @@ class File(models.Model):
         "mail.activity.mixin",
         "abstract.dms.mixin",
     ]
-
     _order = "name asc"
 
     # ----------------------------------------------------------
-    # Database
+    # Fields
     # ----------------------------------------------------------
-
     active = fields.Boolean(
         string="Archived",
         default=True,
         help="If a file is set to archived, it is not displayed, but still exists.",
     )
-
     directory_id = fields.Many2one(
-        comodel_name="dms.directory",
         string="Directory",
+        comodel_name="dms.directory",
         domain="[('permission_create', '=', True)]",
-        context="{'dms_directory_show_path': True}",
+        context={"dms_directory_show_path": True},
```

Comment:
No need to change this.

Reply:
To change what? If you compare with like the v16 or v15 I only removed the outer quotes according to the [official doc](search_panel_select_multi_range) for the context param to be a dict and not a string

Reply:
The link doesn't work to the documentation you mention doesn't work. Can you correct it?

Reply:
Thanks for the link. Well, the option to provide strings is also supported, and it's useful when you need to put a dynamic expression depending on a field value for example, but in this case, it's the same. It was only for reducing the diff of the migration.

Reply:
Does that mean that I can leave as is?